### PR TITLE
Add velux reboot service

### PR DESCRIPTION
**Pull request by pawlizio**
## Proposed change

Documentation for reboot_gateway service added to velux integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/43198
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/velux.markdown [hunk 1]
```diff
@@ -42,6 +42,30 @@ password:
...
+
+```yaml
+automation:
+  alias: KLF reboot on hass stop event
+  description: ''
+  trigger:
+    - event_data: {}
+      event_type: homeassistant_stop
+      platform: event
+  condition: []
```
Comment (frenck) [inline source/_integrations/velux.markdown]: Emtpy condition, can be removed.

```suggestion
```

File: source/_integrations/velux.markdown [hunk 2]
```diff
@@ -42,6 +42,30 @@ password:
...
+
+Reboots the configured KLF 200 Gateway.
+
+There is a problem with the KLF 200 gateway where the connection cannot be established after a restart of Home Assistant, only a manual power off and on fixes this.
+As a workaround, you can use an automation to force a restart of the KLF 200 before exiting Home Assistant, like this:
+
+```yaml
+automation:
+  alias: KLF reboot on hass stop event
+  description: ''
```
Comment (frenck) [inline source/_integrations/velux.markdown]: Empty description can be removed

```suggestion
```

File: source/_integrations/velux.markdown [hunk 3]
```diff
@@ -42,6 +42,30 @@ password:
...
+  trigger:
+    - event_data: {}
+      event_type: homeassistant_stop
+      platform: event
```
Comment (frenck) [inline source/_integrations/velux.markdown]:
```suggestion
    - platform: homeassistant
      event: stop
```

File: source/_integrations/velux.markdown [hunk 4]
```diff
@@ -42,6 +42,30 @@ password:
...
+  condition: []
+  action:
+    - data: {}
+      service: velux.reboot_gateway
```
Comment (frenck) [inline source/_integrations/velux.markdown]:
```suggestion
    - service: velux.reboot_gateway
```

File: source/_integrations/velux.markdown [hunk 5]
```diff
@@ -42,6 +42,30 @@ password:
...
+  mode: single
```
Comment (frenck) [inline source/_integrations/velux.markdown]:
```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/velux.markdown (hunk 1 above)
Comment (pawlizio) [reply source/_integrations/velux.markdown]: Removed condition

File: source/_integrations/velux.markdown (hunk 2 above)
Comment (pawlizio) [reply source/_integrations/velux.markdown]: Added a description

File: source/_integrations/velux.markdown (hunk 3 above)
Comment (pawlizio) [reply source/_integrations/velux.markdown]: Just checked, I suppose the correct event is "shutdown"

File: source/_integrations/velux.markdown (hunk 4 above)
Comment (pawlizio) [reply source/_integrations/velux.markdown]: Removed empty data

**Review by frenck — APPROVED** [review]
Thanks, @pawlizio 👍

**Pull request merged**